### PR TITLE
Make KeyVisualizer disabled by default

### PR DESCRIPTION
**Pull request by smineyev81**
### What problem does this PR solve?
KeyVisualizer sends very expensive queries to TiDB, so we had to disable it manually each time deploy Key Vizualizer https://tidb.support.pingcap.com/servicedesk/customer/portal/4/NAID-2058

### What is changed and how does it work?
Makes KeyVisualizer disabled by default when tidb-dashboard is created first time

### CODE REVIEW

**Comment by baurine** [comment]
hi @smineyev81 , thanks for your contribution, but this change may affect other use cases, we need to investigate its influences or choose a better way to not affect others as possible.

**Comment by smineyev81** [comment]
> hi @smineyev81 , thanks for your contribution, but this change may affect other use cases, we need to investigate its influences, or choose a better way to not affect others as possible.

Would addition of new command line argument (--DisableKeyVisualizer) be a better way of making KeyVisualizer disabled ?

**Comment by baurine** [comment]
> > hi @smineyev81 , thanks for your contribution, but this change may affect other use cases, we need to investigate its influences, or choose a better way to not affect others as possible.
> 
> @baurine , Would addition of new command line argument (--DisableKeyVisualizer) be a better way of making KeyVisualizer disabled ?

Yep, that's a better way.

**Comment by smineyev81** [comment]
> > > hi @smineyev81 , thanks for your contribution, but this change may affect other use cases, we need to investigate its influences, or choose a better way to not affect others as possible.
> > 
> > 
> > @baurine , Would addition of new command line argument (--DisableKeyVisualizer) be a better way of making KeyVisualizer disabled ?
> 
> Yep, that's a better way.

@baurine, command line arg added. Please re-review

**Comment by smineyev81** [comment]
@Renkai, @baurine 
review comments addressed. Could you please re-review

File: cmd/tidb-dashboard/main.go [hunk 1]
```diff
@@ -69,6 +69,7 @@ func NewCLIConfig() *DashboardCLIConfig {
 	flag.BoolVar(&cfg.CoreConfig.EnableExperimental, "experimental", cfg.CoreConfig.EnableExperimental, "allow experimental features")
 	flag.StringVar(&cfg.CoreConfig.FeatureVersion, "feature-version", cfg.CoreConfig.FeatureVersion, "target TiDB version for standalone mode")
 	flag.IntVar(&cfg.CoreConfig.NgmTimeout, "ngm-timeout", cfg.CoreConfig.NgmTimeout, "timeout secs for accessing the ngm API")
+	flag.BoolVar(&cfg.CoreConfig.EnableKeyVisualizer, "keyVisualizer", true, "enable/disable key visualizer(default: true)")
```
Comment (baurine) [inline cmd/tidb-dashboard/main.go:72]: `keyVisualizer` is better to use `keyviz` as flag name to keep consistence with other flags.

**Review by baurine — COMMENTED** [review]
others LGTM

File: cmd/tidb-dashboard/main.go (hunk 1 above)
Comment (smineyev81) [reply cmd/tidb-dashboard/main.go:72]: @baurine,
https://github.com/pingcap/tidb-operator/pull/5536
tidb-operator PR that uses keyVisualizer name as command line arg already merged :(
changing it to keyviz would require another PR to tidb-operator

let me know if you still want me to rename it or it is ok to keep keyVisualizer

Comment (baurine) [reply cmd/tidb-dashboard/main.go:72]: It seems tidb-operator use the camel naming style, so `keyVisualizer` is fine, while tidb-dashboard use the `xx-yy` naming style, I think it's better to keep the same in one project.

Comment (smineyev81) [reply cmd/tidb-dashboard/main.go]: @baurine,
renamed to keyviz
please re-review

**Review by baurine — APPROVED** [review]

**Pull request merged**